ext/mysqlnd/mysqlnd_wireprotocol.c

```diff
@@ -863,9 +859,7 @@ php_mysqlnd_ok_read(MYSQLND_CONN_DATA * conn, void * _packet)
 
 	DBG_RETURN(PASS);
 premature_end:
-	DBG_ERR_FMT("OK packet %zu bytes shorter than expected", p - begin - packet->header.size);
```

Comment:
why are you getting rid of all DBG_ERR_FMT?

Reply:
I guess that they will get logged from error (haven't checked though) but that bytes info can be useful there in some cases.

Reply:
Not all, just a couple that I saw now. Why are you asking? Are you using them? I think we should remove all of them, but it's a lot of work and since it's dead code, there's little harm in keeping them in the code.  If someone ever needs to debug this piece of code, and they use this debug feature, they can add these lines themselves.

Reply:
In case people don't know. `DBG_ERR_FMT` is a macro that can be enabled with some compiler option and when it is on, mysqlnd will spew out debugging information to CLI. If it's off, then the compiler simply discards these lines.